tests/role.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { faker } from '@faker-js/faker/locale/de';
+import { test, expect } from '@playwright/test';
+import { LandingPage } from '../pages/LandingView.page';
+import { LoginPage } from '../pages/LoginView.page';
+import { StartPage } from '../pages/StartView.page';
+import { MenuPage } from '../pages/MenuBar.page';
+import { RoleDetailsViewPage } from '../pages/admin/RoleDetailsView.page';
+import { RoleManagementViewPage } from '../pages/admin/RoleManagementView.page';
+
+const PW = process.env.PW;
+const ADMIN = process.env.USER;
+const FRONTEND_URL = process.env.FRONTEND_URL;
+
+test.describe(`Testfälle für die Anlage von Rollen: Umgebung: ${process.env.UMGEBUNG}: URL: ${process.env.FRONTEND_URL}:`, () => {
+  test('2 Rollen nacheinander anlegen', async ({ page }) => {
+    const Landing = new LandingPage(page);
+    const Startseite = new StartPage(page);
+    const Login = new LoginPage(page);
+    const Menue = new MenuPage(page);
+    const RoleDetailsView = new RoleDetailsViewPage(page);
+    const RoleManagementeView = new RoleManagementViewPage(page);
+
+    const ROLLENNAME1 = 'TAutoR1_' + faker.word.noun();
```

Comment:
Verständnisfrage: Woher kommt "TAutoR1_"?

Reply:
TAuto ist eine Art Prefix, um zu erkennen welche Testdaten durch die Testautomatisierung erstellt wurden. R1 steht für Rolle1, habe ich abgekürzt damit den Rollennamen nicht unnötig lang werden. Wenn es irritiert, kann ich es auch ausschreiben.

Reply:
Musst du nicht ausschreiben, aber vllt ein kurzer Kommentar, dass die Elemente mit Prefix TAuto von Tests angelegt wurden? Am besten in der Readme, weil das ja global gilt, denke ich.

Reply:
erledigt